gtsfm/common/gtsfm_data.py

```diff
@@ -0,0 +1,192 @@
+"""Class to hold the tracks and cameras of a 3D scene.
+
+Authors: Ayush Baid
+"""
+import itertools
+from typing import Dict, List, Optional
+
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler, SfmTrack
+
+import gtsfm.utils.graph as graph_utils
+
+EQUALITY_TOLERANCE = 1e-5
+
+
+class GtsfmData:
+    """Class containing cameras and tracks, essentially describing the complete 3D scene."""
```

Comment:
Thanks for the PR, @ayushbaid . Looks nice. It might be helpful here to explain why we need this class. I.e. why SfmData is not sufficient.

Just if we forget in 6 months or for other people who might want to change it in the future

Reply:
Makes sense. Added it.